pylasu/reflection/reflection.py

```diff
@@ -4,25 +4,30 @@
 
 
 def getannotations(cls):
-    import inspect
-    try:  # On Python 3.10+
-        return inspect.getannotations(cls)
+    try:
+        # https://peps.python.org/pep-0563/
+        return typing.get_type_hints(cls, globalns=None, localns=None)
     except AttributeError:
-        if isinstance(cls, type):
-            return cls.__dict__.get('__annotations__', None)
-        else:
-            return getattr(cls, '__annotations__', None)
+        try:
+            # On Python 3.10+
```

Comment:
Should we mention this in the README?

Reply:
Mention what? The algorithm for getting the annotations?